onnxruntime/python/tools/transformers/onnx_model.py

```diff
@@ -407,6 +407,12 @@ def change_input_output_float32_to_float16(self):
     def convert_model_float32_to_float16(self, cast_input_output=True):
         """ Convert a graph to FLOAT16
         """
+        from packaging.version import Version
+        import onnxconverter_common.float16 as oc
```

Comment:
It is a new dependency. Need update benchmark and notebooks to install the package.

Reply:
Sure